contracts/oracle/ChainlinkOracle.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "../Interfaces/IChainlinkOracle.sol";
+
+/**
+ * @dev The following is a mock Chainlink Price Feed Implementation.
+ *      It is used purely for the purpose of testing.
+ *      All Chainlink price feeds should be wrapped in a Tracer Chainlink Adapter
+ *      to ensure answers are returned in WAD format.
+ *      see contracts/oracle/ChainlinkOracleAdapter.sol.
+ */
+contract ChainlinkOracle is IChainlinkOracle {
+    int256 public price = 100000000;
+    uint8 public override decimals = 8; // default of 8 decimals for USD price feeds in the Chainlink ecosystem
+
+    function latestAnswer() external view override returns (int256) {
+        revert("CO: Deprecated function");
+    }
+
+    function latestTimestamp() external view override returns (uint256) {
+        revert("CO: Deprecated function");
+    }
+
+    function latestRound() external view override returns (uint256) {
+        revert("CO: Deprecated function");
+    }
+
+    function getAnswer(uint256 roundId) external view override returns (int256) {
+        revert("CO: Deprecated function");
+    }
+
+    function getTimestamp(uint256 roundId) external view override returns (uint256) {
+        revert("CO: Deprecated function");
+    }
+
+    function description() external view override returns (string memory) {
+        revert("CO: Deprecated function");
+    }
```

Comment:
@raymogg Should we remove these deprecated calls from the interface? The ChainlinkOracleAdapter no longer supports these.

Reply:
Yeah I think we may as well remove them @OsmanBran since we are staying away from the deprecated api